README.md

```diff
@@ -52,7 +52,7 @@ You can use the [sbt-revolver] plugin to start the server and run it in watch mo
 
 ## Documentation
 
-Learn more on the [ZIO Http homepage](https://github.com/zio/zio-http)!
+Learn more on the [ZIO Http homepage](https://zio.dev/zio-http/)!
```

Comment:
Please don't touch the README.md directly. It is autogenerated.

Reply:
Instead, change `homepage` setting in the `build.sbt` file.

Reply:
Maybe the readme should say that not as a comment but a visible text so I know not to edit it directly in the web UI

Reply:
@nafg Do you have any idea how to notify contributors not to touch the readme?

Reply:
The first line of the rendered readme should say so. Basically what's now a comment should be made visible in the rendered view. Perhaps in a small italicized font.

Reply:
Thank you! Anyway, please let me know if you are willing to finish this work or not.

Reply:
Something came up and I'm really busy for now, so if you could take it over it would be great.

Reply:
I will do that!